src/rules/javascript/deprecated_entities.js

```diff
@@ -27,11 +27,11 @@ export function deprecated_entities(context) {
           node.callee.property.type === 'Identifier' &&
           node.callee.object.type === 'Identifier') {
 
-        let objectName = getNodeReferenceName(context, node.callee.object);
+        let object = getNodeReference(context, node.callee.object);
```

Comment:
Something like this it's probably nice to add something to the var name so we know what object we're talking about later on in the code.